playground/nuxt.config.ts

```diff
@@ -2,13 +2,21 @@ export default defineNuxtConfig({
   modules: ['../src/module', '@nuxt/devtools'],
   ssr: true,
 
-  compatibilityDate: '2024-07-17',
+  build: {
+    transpile: ['@tresjs/core'],
+  },
+
+  future: {
+    compatibilityVersion: 4,
+  },
+
+  compatibilityDate: '2025-03-14',
 
   // for testing purposes: include some nuxt build tests
   nitro: {
     routeRules: {
-      '/page1': { ssr: false }, // <== client rendered page
-      '/page2': { prerender: true }, // <== server SSG page + payload
+      '/page1': { ssr: true }, // <== client rendered page
```

Comment:
Hi @justserdar why the change here, we have this on purpose so we can test the client and the server version of TresCanvas

Reply:
I remember you mentioning the SSR flag from the Discord discussion, it should be set to true, didn't affect the outcome. Being set to false might have been an oversight. Again had the idea the PR was dropped.